azurelinuxagent/common/osutil/systemd.py

```diff
@@ -81,6 +81,6 @@ def get_unit_property(unit_name, property_name):
     #     CPUQuotaPerSecUSec=50ms
     match = re.match("[^=]+=(?P<value>.+)", output)
     if match is None:
-        raise ValueError("Can't find property {0} of {1}", property_name, unit_name)  # pylint: disable=W0715
+        raise ValueError("Can't find property {0} of {1}".format(property_name, unit_name))  # pylint: disable=W0715
```

Comment:
you can remove the pylint suppression now

Reply:
removed